python/lsst/display/firefly/firefly.py

```diff
@@ -84,19 +89,20 @@ def __init__(self, display, verbose=False, host="localhost", port=8080, name="af
         global _fireflyClient
         if not _fireflyClient:
             try:
-                _fireflyClient = FireflyClient.FireflyClient("%s:%d" % (host, port), name)
-                _fireflyClient.launchBrowser()
-                _fireflyClient.addListener(self.__handleCallbacks)
+                _fireflyClient = firefly_client.FireflyClient("%s:%d" % (host, port), name)
+                _fireflyClient.launch_browser()
+                _fireflyClient.add_listener(self.__handleCallbacks)
```

Comment:
Since the new LSST Python style guide hews closely to PEP8, we changed the Firefly API to use PEP8-style names (e.g. no camelCase for function names). Other whitespace changes were introduced by autopep8 as part of the Python 3 porting procedure. It was probably a rookie mistake on my part to include the Python 3 porting together with the more substantial changes in DM-7321.